### PR TITLE
Migrations steps for 1.10 MongoDB `Unexpected exit code 62`

### PR DESCRIPTION
**Pull request by StorytellerCZ**
Add migrations steps in case of `Unexpected mongo exit code 62.`
Based on discussion with @klaussner in #10861

### CODE REVIEW

**Comment by klaussner** [comment]
You could also link to this MongoDB docs page where the update process is descibed in more detail: https://docs.mongodb.com/manual/release-notes/4.2-upgrade-standalone/.

**Comment by filipenevola** [comment]
@StorytellerCZ do you mind making these two suggested changes? 1) if people don't care about local data they can run `meteor reset` this should be the first option. 2) link to mongodb 4.2 release notes

**Comment by StorytellerCZ** [comment]
@larswolter :man_facepalming: Stupid mistake. Fixed.
@klaussner Thanks for the link! Added!
@filipenevola Amended my commit with the changes.

File: History.md [hunk 1]
```diff
@@ -11,7 +11,22 @@
   64-bit Mac, Windows 64-bit, and Linux 64-bit.
 
 ### Migration Steps
-N/A
+* If you get `Unexpected mongo exit code 62. Restarting.` when starting your local
+  MongoDB, you can either restart your project (`meteor reset`)
```
Comment (klaussner) [inline History.md]:
```suggestion
  MongoDB, you can either reset your project (`meteor reset`)
```
😅

**Comment by filipenevola** [comment]
thanks @StorytellerCZ 😉

**Pull request merged**